.github/ISSUE_TEMPLATE/bug_report_v2.yaml

```diff
@@ -0,0 +1,85 @@
+title: '[Bug]: '
+body:
+  - type: markdown
+    id: before-you-start
+    attributes:
+      value: |
+        *[READ THIS]:* to evaluate if you are in the right place?
+        - For issues or feature requests related to __the code in this repository__, file a Github issue.
+        - If this is a __feature request__, make sure the issue title starts with "FR:".
+        - For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/) with the firebase tag.
+        - For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk) google group.
+        - For help troubleshooting your application that does not fall under one of the above categories, reach out to the personalized [Firebase support channel](https://firebase.google.com/support/).
+  - type: input
+    id: operating-system
+    attributes:
+      label: Operating System
+      description: Describe your operating system
+      placeholder: ex. iOS 16.4
+    validations:
+      required: true
+  - type: input
+    id: browser-version
+    attributes:
+      label: Browser Version
+      description: Describe your browser version
+      placeholder: ex. Safari/604.1
+    validations:
+      required: true
+  - type: input
+    id: firebase-sdk-version
+    attributes:
+      label: Firebase SDK Version
+      description: Describe your Firebase SDK Version
+      placeholder: ex. 9.16.0
+    validations:
+      required: true
+  - type: input
+    id: firebase-sdk-product
+    attributes:
+      label: Firebase SDK Product
+      description: Describe your Firebase SDK Product
+      placeholder: ex. Auth
+    validations:
+      required: true
+  - type: textarea
+    id: framework
+    attributes:
+      label: Describe the framework
+      description: Describe the framework your app is built with
+      placeholder: ex. React Native using ExpoGo
+    validations:
+      required: true
+  - type: markdown
+    id: describe-your-problem
+    attributes:
+      value: |
+        'Describe the problem'
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Describe the steps to reproduce
+      placeholder: |
+        ' What happened? How can we make the problem occur?'
+        'This could be a description, log/console output, etc.'
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: javascript
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce-code
+    attributes:
+      label: Code to reproduce issue
+      description: Please copy and paste any relevant code here to reproduce the problem or links to code to reproduce it.
+      render: javascript
+    validations:
+      required: true
```

Comment:
This should probably be right under "Steps to reproduce" and you can probably put a note in the description or placeholder of "steps to reproduce" that if they have any code to reproduce, they can put it in the next field, in case they're worrying about formatting code in the textarea. Does backtick code formatting work in the textarea field? If it does we can just get rid of this field.

Reply:
it does if you have the render attribute set like `render: javascript`

Reply:
Does it format everything they type in the text field into a code block? That doesn't seem like what we want either, if they are going to write a description and then write some code. Is there an option where they can write normal text, and then put backticks around the part they want to be code?